src/core/transport_params.h

```diff
@@ -180,10 +180,10 @@ typedef struct QUIC_TRANSPORT_PARAMETERS {
     uint8_t RetrySourceConnectionIDLength;
 
     //
-    // Pointer to the Version Negotiation transport parameter opaque blob.
+    // Pointer to the version_information transport parameter opaque blob.
     //
-    const uint8_t* VersionNegotiationInfo;
-    QUIC_VAR_INT VersionNegotiationInfoLength;
+    const uint8_t* VersionInfo;
+    QUIC_VAR_INT VersionInfoLength;
```

Comment:
I know this was this way before, but why does `VersionInfoLength` need to be a `QUIC_VAR_INT`? Is this because it's decoded directly from wire format?

Reply:
It doesn't need to be a QUIC_VAR_INT; the length field is no longer sent over the wire, so I can make this any sensible type. I think uint32_t would have the least waste, but uint8_t is probably the realistic limit on the number of versions that could be included.